lib/api/response/question_response.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter_survey/api/response/base/base_response_converter.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'question_response.g.dart';
+
+@JsonSerializable()
+class QuestionResponse {
+  final String id;
+  final String? text;
+  @JsonKey(name: 'display_order')
```

Comment:
What do you think about set it as default in `build.yaml`?
```
targets:
  $default:
    builders:
      json_serializable:
        options:
          field_rename: "snake"
```

More info [here](https://pub.dev/packages/json_serializable#build-configuration)

Reply:
@markgravity agree 👍